docs/runbooks/add-library-site-to-platform.md

```diff
@@ -84,59 +83,43 @@ sites:
 
 The field `plan` defaults to `standard`.
 
-Then continue to provision the a Github repository for the site.
-
-### Step 2: Provision a Github repository
+Now you are ready to sync the site state.
 
-Run `task env_repos:provision` to create the repository.
+### Synchronize site state for all sites
 
-For sites with `plan: webmaster` this also creates a branch `moduletest` which
-represents the environment for testing custom Drupal modules.
+You have made a single additive change to the `sites.yaml` file. It is
+important to ensure that your branch is otherwise up-to-date with `main`,
+as the state you currently have in `sites.yaml` is what will be ensured exists
+in the platform.
 
-#### Create a Lagoon project and connect the GitHub repository
+You may end up undoing changes that other people have done, if you don't have
+their changes to `sites.yaml` in your branch.
 
 Prerequisites:
 
 * A Lagoon account on the Lagoon core with your ssh-key associated (created through
   the Lagoon UI, on the Settings page)
-* The git-url for the sites environment repository (you don't need to create this
-  repository - it will be created by the task below - but the URL must match the
-  Github repository that will be created)
-* A personal access-token that is allowed to pull images from the image-registry
-  that hosts our images.
-* The platform environment name (Consult the [platform environment documentation](https://github.com/danskernesdigitalebibliotek/dpl-platform/wiki/Platform-Environments))
 
-The following describes a semi-automated version of "Add a Project" in
-[the official documentation](https://docs.lagoon.sh/installing-lagoon/add-project/).
+From within `dplsh` run the `sites:sync` task to sync the site state in
```

Comment:
So the only Prerequisity here would be that you are allowed to push (via an ssh key) to the dpl-platform repo?